src/components/ui/LocalizedTime.js

```diff
@@ -147,21 +143,21 @@ export function preset(formatType = 'date-md', dateTimeSeparator = " "){
  * @param {string} [dateTimeSeparator] - Separator between date and time if formatting a date-time. Defaults to ' '.
  */
 export function format(timestamp, formatType = 'date-md', dateTimeSeparator = " ", localize = false, customOutputFormat = null){
-    return display(moment.utc(timestamp), formatType, dateTimeSeparator, localize, customOutputFormat);
+    return display(parseISO(timestamp), formatType, dateTimeSeparator, localize, customOutputFormat);
 }
 
-export function display(momentObj, formatType = 'date-md', dateTimeSeparator = " ", localize = false, customOutputFormat = null){
+export function display(dateObj, formatType = 'date-md', dateTimeSeparator = " ", localize = false, customOutputFormat = null){
     var outputFormat;
     if (customOutputFormat) {
         outputFormat = customOutputFormat;
     } else {
         outputFormat = preset(formatType, dateTimeSeparator);
     }
     if (localize){
-        return momentObj.local().format(outputFormat);
+        formatDate(dateObj, outputFormat, { locale: enUS });
```

Comment:
Can we have this auto-determine what locale the browser is in? We don't want to use enUS if user(s) are not from US.